tests/regression/run-api-with-db.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Use `#!/usr/bin/env bash`

Reply:
I will change all the paths of script interpreter in all the regression script

Reply:
I found it that there are too many files using the old path, so I will open another PR for this